config.py

```diff
@@ -11,11 +12,424 @@
 # folder that all data files are stored in
 DATA_FOLDER = 'data'
 
+# Persistent data storage
+DB_FILE = 'config.db'
```

Comment:
```suggestion
DB_FILE = os.path.join(DATA_FOLDER, 'config.db')
```
Same for in .gitignore, I think the database should go into the data folder